packages/cozy-harvest-lib/src/datacards/FileDataCard.jsx

```diff
@@ -39,8 +46,51 @@ const LoadingFileListItem = ({ divider }) => {
   )
 }
 
+const FileMenu = ({ file, anchorEl, onClose }) => {
+  return (
+    <Menu
```

Comment:
We should use UI's ActionMenu in order to have the right display on mobile device

Reply:
Removed later on.